app/backend/approaches/chatreadretrieveread.py

```diff
@@ -107,17 +131,17 @@ def run(self, history: Sequence[dict[str, str]], overrides: dict[str, Any]) -> A
 
         msg_to_display = '\n\n'.join([str(message) for message in messages])
 
-        return {"data_points": results, "answer": chat_content, "thoughts": f"Searched for:<br>{q}<br><br>Conversations:<br>" + msg_to_display.replace('\n', '<br>')}
+        return {"data_points": results, "answer": chat_content, "thoughts": f"Searched for:<br>{query_text}<br><br>Conversations:<br>" + msg_to_display.replace('\n', '<br>')}
     
     def get_chat_history_as_text(self, history: Sequence[dict[str, str]], include_last_turn: bool=True, approx_max_tokens: int=1000) -> str:
         history_text = ""
         for h in reversed(history if include_last_turn else history[:-1]):
-            history_text = """<|im_start|>user""" + "\n" + h["user"] + "\n" + """<|im_end|>""" + "\n" + """<|im_start|>assistant""" + "\n" + (h.get("bot", "") + """<|im_end|>""" if h.get("bot") else "") + "\n" + history_text
+            history_text = """user:""" + "\n" + h["user"] + "\n\n" + ("""assistant:""" + "\n" + h.get("bot", "") if h.get("bot") else "") + "\n" + history_text
```

Comment:
This has now changed in main, so I think you'll need to merge/rebase and possibly resolve conflicts.